### DISCUSSION TITLE
sales columns in Ads API includes Tax

### DISCUSSION THREAD

**Discussion by Nilanth** [amzn/ads-advanced-tools-docs#270] [Q&A]
Hi

Do the sales columns like sales1d, sales14d, sales7d include only the item price or with tax? Can anyone clarify this?

**Comment by CHERYLKURTENBACH** [comment]
[Completed 2023 TAX ORGANIZER.zip](https://github.com/user-attachments/files/15522300/Completed.2023.TAX.ORGANIZER.zip)
